### ISSUE TITLE
Migrating Single-Tenant to Multi-Tenant

### ISSUE THREAD

**Issue by whwkong** [django-tenants/django-tenants#874]
Currently, I have an instance of a single-tenant app that I'd like to migrate.  

I do not intend to expose urls within the public schema, I only want to create new clients, eg: `client1.xyz.com`, `client2.xyz.com` etc.  

I've created a working greenfield project, with a custom User model, with the following settings:

```python
# public schema
SHARED_APPS = [
    "django_tenants",
    "client",
    "django.contrib.contenttypes",
]

TENANT_APPS = [
    "django.contrib.admin",
    "django.contrib.auth",
    "django.contrib.contenttypes",
    "django.contrib.sessions",
    "django.contrib.messages",
    "django.contrib.staticfiles",
    "django.contrib.sites",
    "django_extensions",
    # 3rd party apps
    "allauth",
    "allauth.account",
    "allauth.socialaccount",
    # my apps
    "apps.users.apps.UsersConfig",
    "myapp", 
]

AUTH_USER_MODEL = "users.User"
```

I do not set `PUBLIC_SCHEMA_URLCONF` as I do not not want to expose any the public schema.

Versions:
```
Django==3.2.16
django-tenants==3.4.7
```

This works.  I can create new tenants, edit the models via the admin panel, etc.  and there are only minimal tables in the public schema.

***

In the [docs for migrating single-tenant to multi-tenant](https://django-tenants.readthedocs.io/en/latest/use.html#migrating-single-tenant-to-multi-tenant), the steps are: 

1. Dump all data from existing public schema `./manage.py dumpdata --all --indent 2 > database.json`
2. Edit settings to connect to new DB
3. Run `manage.py migrate` to create new schemas + tables
4. Flush the db `./manage.py sqlflush | ./manage.py dbshell`
5. Load previous exported data into database

```sh
./manage.py loaddata --format json database.json
```
6. Create a new tenant  `manage.py create_tenant`.  

However, what I want to do is swap steps 5 and 6.

* create a new tenant.
* migrate all the data that was originally from the public schema into the new tenant schema.

Basically run the `loaddata` command within the context of a tenant schema.
Is there a way that I could do this?

**Comment by whwkong** [comment]
Would it be as simple as this?

```sh
manage.py tenant_command loaddata --format json database.json --schema=<tenant_schema>
```

**Comment by mabdullahabid** [comment]
@whwkong I have exactly the same scenario. Did you have any luck figuring this out?

**Comment by tomturner** [comment]
Might be best to ask on stack overflow

**Comment by royroyee** [comment]
@whwkong, did you manage to solve the issue for migrating single-tenant to multi-tenant?
